src/components/mgt-person/mgt-person.ts

```diff
@@ -480,22 +480,22 @@ export class MgtPerson extends MgtTemplatedComponent {
       return;
     }
 
-    // User userId or 'me' query to get the person and image
+    // Use userId or 'me' query to get the person and image
     if (this.userId || this.personQuery === 'me') {
       const graph = provider.graph.forComponent(this);
-      const batch = graph.createBatch();
+      let person = null;
+      let photo = null;
 
       if (this.userId) {
```

Comment:
I actually prefer the batch logic, but I do like that you've moved the logic into separate `*.graph.ts` files. 

Perhaps there is a happy middle, with a function that returns the results of the batch request (person and photo) combined. In fact, if you use the `IDynamicPerson` object from `mgt-person.ts`, you can do it something like this in whichever `*.graph.ts`:
1. Make the Batch request for person and photo.
1. Cast the person as an `IDyanimcPerson`
1. Append the photo to the `person.personImage` property.
1. Return the person.

This could work for both `mgt-person` and `mgt-person-card` as well, and you won't have to make multiple Graph calls 😊

Reply:
Just to clarify - we will have single graph calls to get person, get photo, and a batch call to get both person and photo?

Reply:
Yes, assuming all cases are explicitly used... Add whatever methods make most sense for how we use them. The Graph helper functions aren't intended to be exposed.

Reply:
Makes sense. Easier to read that way too.

Reply:
done